tests/z_api_config_test.c

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+#include <stdio.h>
+#include <string.h>
+
+#include "zenoh.h"
+
+#undef NDEBUG
+#include <assert.h>
+
+void config_client() {
+    const char *peers[] = {"tcp/127.0.0.1", "tcp/192.168.0.1", "tcp/10.0.0.1"};
+    z_owned_config_t config = z_config_client(peers, 3);
+    z_owned_str_t endpoints = zc_config_get(z_loan(config), "connect/endpoints");
+    assert(strcmp(z_loan(endpoints), "[\"tcp/127.0.0.1\",\"tcp/192.168.0.1\",\"tcp/10.0.0.1\"]") == 0);
+    z_drop(&endpoints);
```

Comment:
In Zenoh-Pico, we have it implemented as `z_drop(z_move(endpoints))`.
I know it might seem redundant but by doing so we dont break the rule of being explicit when moving things to the Zenoh API.